core/domain/prod_validation_jobs_one_off.py

```diff
@@ -4818,19 +4830,6 @@ def _get_external_id_relationships(cls, item):
                 user_models.UserSettingsModel, [item.user_id])
         }
 
-    @classmethod
-    def _validate_score_category(cls, item):
```

Comment:
Why are we making this change?

Reply:
@ankita240796 , As per the issue, we need to remove the validation from the suggestion `score_category` so that it doesn't need to belong to a fixed set of strings defined in the constants file. The only validation required is for the score_category to match the exploration category.(which I've added above in the suggestion model validator)
The function `_validate_score_category` above which I removed  validates that score_category to belong to the defined set of values.
As per this comment [link](https://github.com/oppia/oppia/pull/9519#discussion_r440675203)  I removed this. Shouldn't we remove this? cc @DubeySandeep , @nithusha21

Reply:
Ah ok, makes sense. Thanks for explaining.